### ISSUE TITLE
Add documentation section with official purgecss extractors configuration for Vue

### ISSUE THREAD

**Issue by rightaway** [tailwindcss/tailwindcss#1421]
The Tailwind docs have a section about purgecss configuration for extractors here https://tailwindcss.com/docs/controlling-file-size/#writing-purgeable-html, and the Vue docs have it herehttps://purgecss.com/guides/vue.html#usage, but they are different from each other. So as a user of both Tailwind and Vue I don't know whether I should use one or the other or a combination of them somehow.

Could a small section for Vue (and possibly other frameworks) be added to the Tailwind documentation page I linked saying what the official recommendation is for a purgecss extractor when using Tailwind and Vue together?

**Comment by codex-zaydek** [comment]
I know this project makes use of PurgeCSS: https://github.com/mgambill/heroicons-viewer-vue.

Hope that helps!

**Comment by rightaway** [comment]
Also whether we use should `defaultExtractor`, `extractors` or both. Having a few actual examples of the configuration in the docs would really help.

**Comment by devcircus** [comment]
Agree. Ran into problems on the latest version use a class based default extractor. Ended up inlining the extractor to get it to work.
